src/estimators.jl

```diff
@@ -83,14 +82,14 @@ end
     LGBMRegression(; [
         objective = "regression",
         boosting = "gbdt",
-        num_iterations = 10,
+        num_iterations = 100,
         learning_rate = .1,
-        num_leaves = 127,
+        num_leaves = 31,
         max_depth = -1,
         tree_learner = \"serial\",
-        num_threads = Sys.CPU_THREADS,
+        num_threads = 0,
```

Comment:
as per documentation the real cpu cores achieve better speed but the actual default is 0 so changed it across all https://lightgbm.readthedocs.io/en/v3.3.5/Parameters.html#num_threads